firstvoices/backend/tests/test_apis/base_media_test.py

```diff
@@ -15,6 +15,36 @@
 MOCK_THUMBNAIL_LINK = "https://mock_thumbnail_link.com/"
 
 
+def get_site_with_authenticated_member(
```

Comment:
[Optional] Would it be useful to move these functions to `backend/tests/factories/access.py` alongside `get_site_with_member` etc. ? Them seem useful and reusable for future tests that are not specific to media